src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -545,6 +545,9 @@ metaitem.tool.cheat.tooltip=Also an Infinite Energy Source
 metaitem.tool.scanner.name=Portable Scanner
 metaitem.tool.scanner.tooltip=Tricorder
 
+metaitem.rebreather.name=Rebreather
+metaitem.rebreather.tooltip=Allows breathing under water in expense of consuming energy
```

Comment:
**Allows breathing underwater at the expense of energy**

"underwater" is becoming more properly accepted for all use cases in English, but its debatable.
"at the expense" is always more correct than "in expense"
"consuming" is just redundant

Reply:
Tooltip tweaked as requested